sandbox/lua/decoders/linux_procstat.lua

```diff
@@ -0,0 +1,109 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+
+Parses a payload containing the contents of a `/proc/stat` file. This also
+tries to obtain the TickerInterval of the input it recieved the data from, by
+extracting it from a message field named `TickerInterval`.
+
+Config:
+
+- payload_keep (bool, optional, default false)
+    Always preserve the original log line in the message payload.
+
+*Example Heka Configuration*
+
+.. code-block:: ini
+
+    [ProcStats]
+    type = "FilePollingInput"
+    ticker_interval = 1
+    file_path = "/proc/stat"
+    decoder = "ProcStatDecoder"
+
+    [ProcStatDecoder]
+    type = "SandboxDecoder"
+    filename = "lua_decoders/linux_procstat.lua"
+
+*Example Heka Message*
```

Comment:
i added the index entries (index_count, index) ... but thses will be removed i hope.